### DISCUSSION TITLE
Saves do not work unless I open the application and exit it forcefully

### DISCUSSION THREAD

**Discussion by garbaj-sys** [bucanero/apollo-ps4#175] [Q&A]
I have been having a strange problem ever since a I did a data migration from HDD to SSD in PS4 9.00. I made a backup of my saves using Apollo Save Tool. Then I restored (copied) all the saves using Apollo Save Tool to the new SSD.

The PS4 system on the Settings menu can properly identify each save file and even show the correct game. However, these restored save are seen as corrupted in-game . I found a "workaround" that If I launch Apollo Save Tool and exit forcefully (using the home button), the saves are properly recognized by games. Therefore I have to always launch the Apollo Save tool once per cold-boot for the save games to work. I notice that I also see a popup saying that "PS4 Saves patches have been applied to memory" on the startup of the application.  If i exit normally through the app, the save files do not work again. I also see that  I  get the notification that "save patches are removed from memory" when exiting the app this way. So basically I think all my saves work only if the save patches are applied to memory from what I have seen. 

I am so confused to why I have this problem and I don't know how i fix the save files without the need to boot Apollo tool everytime, because I think the current workaround is maybe not the intended way to use Apollo Tool. 

EDIT: Some additional info as well which might be relevant. The saves on the HDD were stored in a fake-activated account (the fake PSN account you create offline with the exploit). Now when i moved the saves to the new SSD I decided to move everything to a local account (instead of creating another fake PSN account).

**Comment by bucanero** [comment]
Saves remain tied to the fake account ID you created on the previous setup. The best alternative is to activate the new system with the same fake account ID.
